source/guides/clear/performance.rst

```diff
@@ -0,0 +1,236 @@
+.. _performance:
+
+Performance
+###########
+
+|CL-ATTR| is built with optimizations across the whole stack for improved
+performance. |CL| achieves its performance through a variety of design decisions
+and software building techniques.
+
+.. contents:: :local:
+   :depth: 1
+
+Overview
+********
+
+The |CL|  philosophy is to do everything with performance in mind. The |CL| team

```

Comment:
Please transform this  section into bullets so it's easier to scan. Something like: 
```
|CL|:  
* Approaches performance holistically ...
* Optimizes for runtime performance.... 
```

Reply:
Good improvement :)